src/pages/ManageValidator/LeftManage/ValidatorBlueSection/ValidatorBlueSection.tsx

```diff
@@ -1,16 +1,12 @@
 import { YStack } from 'tamagui'
 
 import ValidatorInfo from './ValidatorInfo'
-import DepositStatus from './DepositStatus/DepositStatus'
+import DepositStatus from '../DepositStatus/DepositStatus'
 import ValidatorBalance from './ValidatorBalance'
 
 const ValidatorBlueSection = () => {
   return (
-    <YStack
-      backgroundColor="#F4F6FE"
-      space={'$3'}
-      style={{ padding: '30px 15px 15px 30px' }}
-    >
+    <YStack backgroundColor="#F4F6FE" space={'$2'}>
```

Comment:
This is a change introduced by @RadoslavDimchev [here](https://github.com/nimbus-gui/nimbus-gui/commit/e76b4f00a2050b30c728bc8878207adc1af86819) why do you revert it?